foosball/games/models.py

```diff
@@ -22,6 +30,8 @@ class Game(TimeStampedModel):
         help_text=_("Time when the game was played")
     )
 
+    table = models.ForeignKey(Table, blank=True, null=True, related_name="games", verbose_name=_("table"))
```

Comment:
Ideally this would not be nullable - otherwise we'll end up with games without a table. Unless that is what we want :) To track games in a pub for example..